agent/consul/kvs_endpoint_test.go

```diff
@@ -475,6 +478,9 @@ func TestKVSEndpoint_List_ACLDeny(t *testing.T) {
 			}
 		}
 	}
+	if !dirent.QueryMeta.ResultsFilteredByACLs {
+		t.Fatal("ResultsFilteredByACLs should be true")
+	}
 }
 
 func TestKVSEndpoint_List_ACLEnableKeyListPolicy(t *testing.T) {
```

Comment:
could we add a test here (or somewhere else) to test that `ResultsFilterdByACLs` is set for the `List` endpoint as expected?

Reply:
I think this is covered here:

https://github.com/hashicorp/consul/blob/b5f81dd0c0f8361e986dbb86fde96fe5644f50da/agent/consul/kvs_endpoint_test.go#L481-L483